### ISSUE TITLE
/_matrix/federation/v1/version returns the "404 page not found"

### ISSUE THREAD

**Issue by fax1ty** [matrix-org/dendrite#2868]
### Background information

- **Dendrite version or git SHA**: f10c6f26
- **Monolith or Polylith?**: Monolith 
- **SQLite3 or Postgres?**: Postgres
- **Running in Docker?**: Yes
- **`go version`**: 1.19.2
- **Client used (if applicable)**: Third Room

### Description

- **What** is the problem: /_matrix/federation/v1/version returns the `404 page not found` message
- **Who** is affected: Any user who join the world
- **How** is this bug manifesting: [Remote objects are not synchronized](https://github.com/matrix-org/thirdroom/issues/418) in the created world. The federation tester [returns an error](https://github.com/matrix-org/matrix-federation-tester/issues/130)
- **When** did this first appear: As soon as I was able to start the server

### Steps to reproduce

- Run Dendrite inside a Docker [[configuration]](https://hastebin.com/kibusikoqa.less)
- Configure a reverse proxy [[configuration]](https://hastebin.com/huwocemare.nginx)
- Configure a well-known files [[ex. server]](https://mercdev.com/.well-known/matrix/server) / [[ex. client]](https://mercdev.com/.well-known/matrix/client)
- Join the room [[ex.]](https://meta.mercdev.com/world/#test:mercdev.com) / [[credentials]](https://hastebin.com/uluxecubax.apache)

<!--
Describe how what happens differs from what you expected.

If you can identify any relevant log snippets from server logs, please include
those (please be careful to remove any personal or private data). Please surround them with
``` (three backticks, on a line on their own), so that they are formatted legibly.

Alternatively, please send logs to @kegan:matrix.org or @neilalexander:matrix.org
with a link to the respective Github issue, thanks!
-->

**Comment by neilalexander** [comment]
What is the value of `disable_federation` in your Dendrite config file?

**Comment by fax1ty** [comment]
@neilalexander [Configuration](https://hastebin.com/ladusajufu.yaml)
It says `true` here
But is this a problem of the players lack of synchronization?

**Comment by neilalexander** [comment]
If you've disabled federation then of course federation will not work.

**Comment by fax1ty** [comment]
@neilalexander 

Yes, the federation check is now successful, but the remote objects are still not visible
![image](https://user-images.githubusercontent.com/26363214/200550271-ef7636ef-2644-42af-a9e7-d8687198437d.png)
I noticed that we are not creating an RTC connection. Does Third Room use UDP via RTC to communicate between client and server?
![image](https://user-images.githubusercontent.com/26363214/200550190-387fdb27-9b21-4450-9e16-99fc3d71caa5.png)

**Comment by neilalexander** [comment]
Afraid I have no idea how Third Room works so I can't help with that, only thing I can suggest is that you try configuring some TURN servers in your Dendrite config.

**Comment by MadLittleMods** [comment]
Third room discussion here: https://matrix.to/#/!rzmCByrDIhEbURWmyW:matrix.org/$TniqzCvft-LYmveG5lu9cXSKFZ45y1VwgtBIfLw5DU8?via=matrix.org&via=element.io&via=tchncs.de

> You do need a TURN server set up. They are processed by WebRTC (although it’s currently using TCP not UDP)

See https://github.com/matrix-org/synapse/blob/develop/docs/turn-howto.md ([permalink](https://github.com/matrix-org/synapse/blob/a5fcdea090c2396c30dd07c357ce4d9c90004c34/docs/turn-howto.md))

**Comment by S7evinK** [comment]
The documentation for Synapse can be used for Dendrite as well, using this setup for Dendrite and tested it successfully.

**Comment by spaetz** [comment]
Given that this is a user config issue, this one can probably be closed?
